Missionframework/functions/fn_addActionsPlayer.sqf

```diff
@@ -370,4 +370,20 @@ _player addAction [
     "
 ];
 
+// Drop crate
+_player addAction [
+    "<t color='#FFFF00'>", localize "STR_ACTION_CRATE_DROP", "</t>"] joinString "",
```

Comment:
```suggestion
    ["<t color='#FFFF00'>", localize "STR_ACTION_CRATE_DROP", "</t>"] joinString "",
```

Reply:
I'm wondering why it doesn't throw any errors while testing 🤔